llx/builtin_version.go

```diff
@@ -37,6 +37,10 @@ func NewVersion(s string) Version {
 	}
 }
 
+func (v *Version) IsSemver() bool {
+	return v.Version != nil && v.epoch == 0
```

Comment:
I am not sure I understand why epoch 0 is an indicator of semver? Should we not try to parse it an return false when semver.NewVersion returns an error?